plugins/collapseGroups.js

```diff
@@ -68,11 +71,17 @@ export const fn = () => {
           node.children.length === 1
         ) {
           const firstChild = node.children[0];
+          let nodeHasFilter = node.attributes.filter ? true : false;
```

Comment:
This could probably be simplified to `let nodeHasFilter = node.attributes.filter`

Reply:
If we were to change this, I'd prefer this so it's explicitly a boolean:

```suggestion
          let nodeHasFilter = !!node.attributes.filter;
```

Naming a variable after a boolean when it's actual a truthy/falsey makes the code confusing imo.

Reply:
TypeScript doesn't like `let nodeHasFilter = node.attributes.filter`.

Reply:
Also `Boolean(expr)` is always an option